build.cake

```diff
@@ -122,10 +124,14 @@ Task("BuildEventBuilder")
     .Does (() =>
 {
     var solution = "./src/EventBuilder.sln";
+    Information("Building {0} using {1}", solution, msBuildPath);
 
     NuGetRestore(solution, new NuGetRestoreSettings() { ConfigFile = "./src/.nuget/NuGet.config" });
 
-    MSBuild(solution, new MSBuildSettings()
+    MSBuild(solution, new MSBuildSettings() {
+            ToolPath = msBuildPath,
```

Comment:
Had to specify the location of msbuild (an oversight, we do this as part of the `buildreactiveui` step and forgot to add it in here)